<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1566</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Трудовая книжка</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" rowspan="3">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>222</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>222</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="5" rowspan="3" colspan="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Сведенья о работе</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>№ Записи</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Сведенья</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Документ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Удаление</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="7" colspan="2">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Распознанные слова</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2" rowspan="3" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Профессия/специальность</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Удалить книжку</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Образование</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Дата рождения</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Оцифровать главную страницу</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Дата выдачи книжки</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Выгрузить в JSON</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Номер трудовой книжки</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Серия трудовой книжки</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="8">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="6">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="7">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
